src/applications/personalization/profile/components/direct-deposit/BankInfoForm.jsx

```diff
@@ -6,93 +6,117 @@ import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
 
 import { ACCOUNT_TYPES_OPTIONS } from '../../constants';
 
-export const schema = {
-  type: 'object',
-  properties: {
-    routingNumber: {
-      type: 'string',
-      pattern: '^\\d{9}$',
-    },
-    accountNumber: {
-      type: 'string',
-      pattern: '^\\d{1,17}$',
-    },
-    accountType: {
-      type: 'string',
-      enum: Object.values(ACCOUNT_TYPES_OPTIONS),
+export function makeFormProperties(prefix) {
+  return {
+    routingNumber: `${prefix}RoutingNumber`,
+    accountNumber: `${prefix}AccountNumber`,
+    accountType: `${prefix}AccountType`,
+  };
+}
+
+function makeSchemas(prefix) {
```

Comment:
This method, making it possible to make different versions of the form schemas, is the crux of this PR.